src/components/pages/search.yml

```diff
@@ -2,7 +2,7 @@ title:
   en: Title
   de: Titel
 udateDate:
-  en: Udate date
+  en: Update date
```

Comment:
Nice spot, maybe udate the key as well :D

Reply:
Ah true, corrected, thanks!